ding/utils/lock_helper.py

```diff
@@ -97,23 +109,63 @@ def get_rw_file_lock(name: str, op: str):
         return lock.gen_wlock()
 
 
+import fcntl
```

Comment:
remove this import here, it is available only in windows, you can refer to the beginning of this file for more details